docker-compose.yaml

```diff
@@ -82,7 +82,7 @@ services:
       - local
 
   backend:
-    image: mzdotai/lumigator:latest
+    image: mzdotai/lumigator:v0.1.0-alpha
```

Comment:
Generally I would have suggested we make some sort of "latest-release" tag so that we don't need to update this on every new release, but this is fine for the time being 👍

Reply:
Sounds good! To clarify, if you run `start-lumigator-build` it will run with the latest version on the main branch, not necessarily the latest release? By release I mean the latest published 'release'/ git release tag version of the code, which is not the same as the latest on the main branch.